rancher/post_install.md

```diff
@@ -15,13 +14,18 @@ metadata:
     kubernetes.io/ingress.class: traefik
 spec:
   rules:
-  - http:
+  - host: example.com
```

Comment:
Can you test with host : rancher.<your-cluster-id>.k8s.civo.com ? if that works please change it

Reply:
Tested it even with other dynamic DNS providers and I can confirm this ingress configuration should just work (both Traefik and NGINX).